index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+    <!-- <link rel="stylesheet" href="https://kit.fontawesome.com/058c29e44e.css" crossorigin="anonymous"> -->
+</head>
+<body>
+    <header class="header">
+        <div class="logo-container">
+          <i class="material-symbols-rounded">menu</i>
```

Comment:
A hyperlink for the menu icon will be more appropriate.

Reply:
Great work. You seem to be adept at the use of inline comments 💪. However, just like I mentioned in my previous comment, it is always best to keep your review comments as friendly suggestions 👍. 

Also, the receiver of your review could really benefit from the reason why a hyperlink for the menu icon will be more appropriate. In that light, it is a good thing to always try to explain your review comments as much as possible and give as much context as possible. 👍